src/main/java/HW_5/myClass/Person.java

```diff
@@ -0,0 +1,47 @@
+package HW_5.myClass;
+
+public class Person {
+    String name;
+    String surname;
+    int age;
+
+     public Person (String name, String surname, int age){
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "name='" + name + '\'' +
+                ", surname='" + surname + '\'' +
+                ", age=" + age +
+                '}';
```

Comment:
Please remove unused methods in Person class

Reply:
fixed